File: parantier-api/src/main/java/com/mapo/palantier/issue/domain/IssueChecklist.java
package com.mapo.palantier.issue.domain;

import lombok.Getter;
import lombok.Builder;

import java.time.LocalDateTime;

@Getter
@Builder
public class IssueChecklist {
    private Long id;
    private Long issueId;
    private String content;
    private Boolean checked;
    private Integer orderNum;
    private LocalDateTime createdAt;
}

File: parantier-front/src/entities/issue/types/issueChecklist.ts
export interface IssueChecklist {
  id: number
  issueId: number
  content: string
  checked: boolean
  orderNum: number
  createdAt: string
}

export interface CreateChecklistRequest {
  content: string
  orderNum?: number
}

export interface UpdateChecklistRequest {
  content: string
  checked: boolean
  orderNum: number
}
